src/pages/settings/Subscription/CardSection/utils.ts

```diff
@@ -42,11 +42,9 @@ function getBillingStatus({translate, accountData, purchase}: GetBillingStatusPr
 
     const purchaseAmount = purchase?.message.billableAmount;
     const purchaseCurrency = purchase?.currency;
-    const isBillingTypeProper = purchase?.message.billingType === 'failed_2018';
```

Comment:
Lets re-add this condition

> We should validate that purchaseList.message.billingType is failed_2018 before displaying the Your April 3rd, 2024 charge of $41.58 could not be processed. portion of the subtitle, just to be sure.

https://github.com/Expensify/App/issues/58067#issue-2905698820